<?php

if(!@include_once(__ROOT . 'app/core/auth/validation.php')) throw new Exception('1');
if(!@include_once(__ROOT . 'app/core/auth/sigin.php')) throw new Exception('2');
if(!@include_once(__ROOT . 'app/core/auth/signup.php')) throw new Exception('3');

?>